client/src/App.js

```diff
@@ -1,28 +1,12 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, {Component} from 'react'
+import HomeContainer from './containers/HomeContainer'
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
-  }
+	render() {
```

Comment:
please use spaces (2) and not tabs :) 

![image](https://user-images.githubusercontent.com/883348/50512554-eed5c200-0a92-11e9-9b7c-389d9e00ef73.png)